metadata_service/proxy/atlas_proxy.py

```diff
@@ -706,6 +734,47 @@ def get_table_by_user_relation(self, *, user_email: str, relation_type: UserReso
                 cluster=res['cluster'],
                 schema=res['db'],
                 name=res['table']))
+        return results
+
+    def _get_resource_owned_by_user(self, user_id: str, resource_type: str) \
+            -> List[Union[PopularTable, DashboardSummary, Any]]:
+        """
+        ToDo (Verdan): Dashboard still needs to be implemented.
+        Helper function to get the resource, table, dashboard etc owned by a user.
+        :param user_id: User ID of a user
+        :param resource_type: Type of a resource that returns, could be table, dashboard etc.
+        :return: A list of PopularTable, DashboardSummary or any other resource.
+        """
+        owned_by_resources = list()
+        user_entity = self._driver.entity_unique_attribute(self.USER_TYPE, qualifiedName=user_id).entity
+
+        if not user_entity:
+            LOGGER.exception(f'User ({user_id}) not found in Atlas')
+            raise NotFoundException(f'User {user_id} not found.')
+
+        resource_guids = list()
+        for item in user_entity[self.REL_ATTRS_KEY].get('ownerOf') or list():
```

Comment:
could that just be `.get('ownerOf', []) `?

Reply:
nope, that's different than this.

Reply:
how is it different ? `or list()` is just a fallback if `ownerOf` doesn't exist

Reply:
I love talking about Python 😄 
<img width="432" alt="Screenshot 2020-08-25 at 15 41 21" src="https://user-images.githubusercontent.com/25360476/91181704-8d437c00-e6e9-11ea-9d09-67b47556688c.png">

Reply:
that I understand, but not sure why would client return `"gamma": None`, doesn't seem like sensible thing - if there are no results then should be empty list. Anyways if that's actually the case then fine leaving it as is.